src/NLog/Targets/WebServiceTarget.cs

```diff
@@ -250,6 +250,45 @@ protected override void DoInvoke(object[] parameters, AsyncContinuation continua
             }
         }
 
+        /// <summary>
+        /// Builds the URL to use when calling the web service for a message, depending on the WebServiceProtocol.
+        /// </summary>
+        /// <param name="parameterValues"></param>
+        /// <returns></returns>
+        private Uri BuildWebServiceUrl(object[] parameterValues)
+        {
+            if (this.Protocol != WebServiceProtocol.HttpGet)
+            {
+                return this.Url;
+            }
+            
+            //if the protocol is HttpGet, we need to add the parameters to the query string of the url
+            var queryParameters = new StringBuilder();
+            string separator = string.Empty;
+            int i = 0;
+            foreach (MethodCallParameter parameter in this.Parameters)
+            {
+                queryParameters.Append(separator);
+                queryParameters.Append(parameter.Name);
+                queryParameters.Append("=");
+                queryParameters.Append(UrlHelper.UrlEncode(Convert.ToString(parameterValues[i], CultureInfo.InvariantCulture), true));
+                separator = "&";
+                i++;
+            }
+
+            var builder = new UriBuilder(this.Url);
+            if (builder.Query != null && builder.Query.Length > 1)
```

Comment:
Yes. I know it looks weird, but that's how the property is supposed to be used : [UriBuilder.Query doc](https://msdn.microsoft.com/en-us/library/system.uribuilder.query.aspx). Is it OK to include a link to this page in the comment ?